### PR TITLE
add CI action and fixed pylint error

### PR DESCRIPTION
**Pull request by FateScript**
* add CI action, now we support pylint, GPU check is not included.
* fixed pylint error in official.vision

### CODE REVIEW

File: official/vision/classification/shufflenet/model.py [hunk 1]
```diff
@@ -97,6 +97,8 @@ def forward(self, old_x):
             x_proj = old_x
             x = old_x
             return F.concat((self.branch_proj(x_proj), self.branch_main(x)), 1)
+        else:
+            raise ValueError("use strdie 1 or 2, current stride {}".format(self.stride))
```
Comment (wjfwzzc) [inline official/vision/classification/shufflenet/model.py:101]: there is a typo: `strdie` -> `stride`

File: README.md [hunk 2]
```diff
@@ -1,3 +1,5 @@
+![pylint](https://github.com/FateScript/Models-1/CI/badge.svg)
```
Comment (wjfwzzc) [inline README.md:1]: url should be changed.

Comment (FateScript) [reply README.md]: done

**Comment by wjfwzzc** [comment]
LGTM.

**Pull request merged**